scripts/build_utils.sh

```diff
@@ -1373,6 +1377,11 @@ setup_rpm_build_deps() {
         # before EPEL8 and PowerTools provide all dependencies, we use sepia for the dependencies
         $SUDO dnf config-manager --add-repo http://apt-mirror.front.sepia.ceph.com/lab-extras/8/
         $SUDO dnf config-manager --setopt=apt-mirror.front.sepia.ceph.com_lab-extras_8_.gpgcheck=0 --save
+
+	$SUDO dnf install -y epel-release || sudo dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1).noarch.rpm
+	$SUDO dnf install -y https://apache.jfrog.io/artifactory/arrow/centos/$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1)/apache-arrow-release-latest.rpm
+	$SUDO dnf --enablerepo=powertools install -y snappy-devel
+	$SUDO dnf config-manager --set-enabled epel || :
```

Comment:
i don't think these are needed.

Reply:
snappy is required for arrow build (while testing that on centos8-container) 
in case it is not needed for the shaman build, I will remove that.

generally, in order to have correct results, it needs a suitable environment to test that.
currently, I'm using a container(per platform) to test that.

Reply:
please refer to https://github.com/ceph/ceph/blob/master/ceph.spec.in.

> 
> generally, in order to have correct results, it needs a suitable environment to test that.
> currently, I'm using a container(per platform) to test that.

i am not sure i follow you.. how is this related to the change here?

Reply:
> please refer to https://github.com/ceph/ceph/blob/master/ceph.spec.in.
> 
> > generally, in order to have correct results, it needs a suitable environment to test that.
> > currently, I'm using a container(per platform) to test that.
> 
> i am not sure i follow you.. how is this related to the change here?

its not relate to change here, its a general statement(any change in this PR)